src/test/java/io/debezium/connector/yugabytedb/YugabyteDBPublicationReplicationTest.java

```diff
@@ -0,0 +1,193 @@
+package io.debezium.connector.yugabytedb;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+import io.debezium.connector.yugabytedb.common.YugabyteDBContainerTestBase;
+import io.debezium.connector.yugabytedb.common.YugabytedTestBase;
+import io.debezium.connector.yugabytedb.connection.YugabyteDBConnection;
+
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.*;
+import org.yb.client.GetDBStreamInfoResponse;
+import org.yb.client.YBClient;
+
+import io.debezium.config.Configuration;
+import io.debezium.connector.yugabytedb.transforms.YBExtractNewRecordState;
+import io.debezium.transforms.ExtractNewRecordStateConfigDefinition;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class YugabyteDBPublicationReplicationTest extends YugabytedTestBase {
```

Comment:
The class to be extended would be `YugabyteDBContainerTestBase`

Also, put a javadoc comment telling what this test is for and what is the minimum service version required for it to run.